src/confopt/profile/base.py

```diff
@@ -386,34 +709,143 @@ def configure_partial_connector(self, **kwargs) -> None:  # type: ignore
             self.configure_searchspace(k=kwargs["k"])
 
     def configure_trainer(self, **kwargs) -> None:  # type: ignore
+        """Configure the trainer component for the supernet.
+
+        Args:
+            **kwargs: Arbitrary keyword arguments. Possible keys include:
+
+                lr (float): Learning rate for the optimizer.
+
+                arch_lr (float): Learning rate for architecture parameters.
+
+                epochs (int): Number of training epochs.
+
+                optim (str): Optimizer type. Can be 'sgd', 'adam', etc.
+
+                arch_optim (str): Optimizer type for architecture parameters.
+
+                optim_config (dict): Additional configuration for the optimizer.
+
+                ...
```

Comment:
It would be useful to list all the possible keywords here, because otherwise the user has to go through the code to see what all are used. The exhaustive list can be found in the `Experiment` class.

Reply:
This isn't actually done?